enhancements/network/egress-router.md

```diff
@@ -0,0 +1,100 @@
+---
+title: OVN Egress Router Support
+authors:
+  - "@dmellado"
+  - "@jdesousa"
+reviewers:
+  - "@squeed"
+  - "@danwinship"
+approvers:
+  - "@knobunc"
+creation-date: 2020-08-26
+last-updated: 2020-08-26
+status: provisional
+---
+
+# OVN Egress Router Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Egress traffic is traffic going from OpenShift pods to external systems,
+outside of OpenShift. There are two main options for enabling egress traffic:
+Allow access to external systems from OpenShift physical node IPs or use egress
+router.
```

Comment:
"There are two main options for enabling _access control of_ egress traffic". Except also that's not true because we have `EgressFirewall` and egress IPs as well.

This proposal is supposed to be mostly about the _new_ egress router design, so I think you can just assume that the reader knows about the existing egress router and its use cases, and focus more on how we're changing it.